packages/server-api/src/index.ts

```diff
@@ -170,4 +172,9 @@ export interface ServerAPI extends PluginServerApp {
     ) => void
   }) => void
   getSerialPorts: () => Promise<Ports>
+  getCourse: () => CourseInfo
+  setCourse: (
+    dest: (Destination & { arrivalCircle?: number }) | null
+  ) => Promise<void>
+  setRoute: (dest: RouteDest | null) => Promise<void>
```

Comment:
The implementations call `setDestination` and `activateRoute` and the http API uses PUTs to similar paths. Why now use the same terminology here? To me this obfuscates things needlessly - why would the API exposed to plugins use different naming than the internal one? Why not use the exact same naming?

Reply:
Have aligned interface method names.

Reply:
Is it better in the long run to namespace these methods with the API they are related to?

If so then `getCourse()` could be renamed to just `get()`

For example:
```
app.course.get()

app.course.destination(..)

app.course.activeRoute(...)
```

Reply:
😓 sorry, i probably should have been clearer with my intent: I think these methods should be like they were in the internal API:
- "set destination" => setDestination(...)
- "activate route (and set next point, intermediate destination)" => activateRoute(...)
in HTTP the `set` is the http verb, but in ts without it I find the semantics is totally obscure: is this causing a side effect or is the parameter needed for getting a value?

As to your second comment: from developer perspective code is more readable and autocomplete works better without extra nesting.

Reply:
So hopefully I have interpreted this correctly in the latest commit.

The interface methods have now been exposed to `app` in alignment with the internal method called by the API endpoint.

```
    appCopy.getCourse = () => {
      return courseApi.getCourse()
    }
    appCopy.setDestination = (
      dest: (Destination & { arrivalCircle?: number }) | null
    ) => {
      return courseApi.destination(dest)
    }
    appCopy.activateRoute = (dest: RouteDest | null) => {
      return courseApi.activeRoute(dest)
    }
```